view/template/dashboard.hbs

```diff
@@ -5,63 +5,77 @@
             Dashboard
         </li>
     </ol>
-    <div class="col-lg-4 col-md-12">
-        <div class="card">
+    <div class="col-sm-12">
+      <div class="col-sm-12 col-md-5 col-lg-6 col-xl-6">
+        <div class="col-sm-8 col-md-12 col-lg-12 col-xl-12">
```

Comment:
Grids inside of grids inside of grids?
Got some [Inception](http://www.imdb.com/title/tt1375666/) going on here. :smiley: 
I get why two levels could be needed.
What is the third level doing?